src/app/csv-collection/csv-collection.service.ts

```diff
@@ -53,7 +54,7 @@ export class CsvCollectionService {
     httpOptions.params = httpParams;
     return this.http.get<any>(this.csvCollectionUrl + '/search/findByNameContainingIgnoreCase', httpOptions).pipe(
       map((result: any) => {
-        result.csvCollections = result._embedded.csvCollections;
+        result.data = result._embedded.csvCollections;
```

Comment:
This will break the "filter by name" in the csv collections list page (same issue for other data types)